packages/terra-avatar/src/variants/Facility.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import memoize from 'memoize-one';
+import 'terra-base/lib/baseStyles';
+import Utils from '../common/AvatarUtils';
+import styles from '../common/Avatar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Specifies the alternative text for the image.
+   */
+  alt: PropTypes.string.isRequired,
+  /**
+   * Sets the background color. Defaults to `auto`. Accepted color variants are theme specific.
+   * One of: `'auto'`, `'neutral'`, `'one'`, `'two'`, `'three'`, `'four'`, `'five'`, `'six'`, `'seven'`, `'eight'`, `'nine'`, `'ten'`.
+   */
+  color: PropTypes.oneOf(['auto', 'neutral', 'one', 'two', 'three', 'four',
+    'five', 'six', 'seven', 'eight', 'nine', 'ten']),
+  /**
+   * Value used for the hash function when color is set to `auto`. If not provided, hash function utilizes alt.
+   */
+  hashValue: PropTypes.string,
+  /**
+   * The image to display.
+   */
+  image: PropTypes.string,
+  /**
+   * Whether to hide avatar from the accessiblity tree.
+   */
+  isAriaHidden: PropTypes.bool,
+  /**
+   * Overrides the default width.
+   */
+  size: PropTypes.string,
+};
+
+const defaultProps = {
+  color: 'auto',
+  hashValue: undefined,
+  image: undefined,
+  isAriaHidden: false,
+  size: undefined,
+};
+
+const ImageComponent = memoize(Utils.generateImage);
+
+const Facility = ({
+  alt,
+  color,
+  hashValue,
+  image,
+  isAriaHidden,
+  size,
+  ...customProps
+}) => {
+  let colorVariant;
+  let facilityContent;
+  if (image) {
+    colorVariant = '';
+    facilityContent = ImageComponent(image, alt, isAriaHidden, Utils.AVATAR_VARIANTS.FACILITY);
+  } else {
+    colorVariant = Utils.setColor(alt, color, hashValue);
+    facilityContent = Utils.generateImagePlaceholder(alt, isAriaHidden, Utils.AVATAR_VARIANTS.FACILITY);
+  }
+
+  const attributes = { ...customProps };
+  const facilityClassNames = cx([
+    'avatar',
+    `${colorVariant}`,
+    attributes.className,
+  ]);
+
+  return (
+    <div {...attributes} className={facilityClassNames} style={{ fontSize: size }}>
```

Comment:
I'd update this to something like this below to allow people to set additional inline styles if they need to:

```jsx
const customStyles = size ? Object.assign({ fontSize: size}, style) : style;

return (
  <div className={facilityClassNames} style={customStyles} {...attributes}>
    {avatarContent}
  </div>
);
```

Reply:
updated https://github.com/cerner/terra-core/pull/2061/commits/ba380de6fb977aeaee0a2ff74e5472c2c8413886